cmd/dlv/cmds/commands.go

```diff
@@ -327,6 +327,8 @@ Currently supports linux/amd64 and linux/arm64 core files, windows/amd64 minidum
 		},
 		Run: coreCmd,
 	}
+	core := false
+	coreCommand.Flags().BoolVarP(&core, "core", "c", false, "")
```

Comment:
There should be a comment here saying that the core flag is unused and this only exists so that `dlv core` can be used by coredumpctl.